### ISSUE TITLE
Natural can't find Lodash in create-react-app

### ISSUE THREAD

**Issue by sixmoraltales** [NaturalNode/natural#517]
When trying to implement Natural in a simple create-react-app (which by default has lodash npm package installed), I get the following error when running the sample code from the readme:

TypeError: Cannot read property 'without' of undefined
WordTokenizer.push../node_modules/natural/lib/natural/tokenizers/regexp_tokenizer.js.RegexpTokenizer.tokenize
node_modules/natural/lib/natural/tokenizers/regexp_tokenizer.js:48
```

  45 | 
  46 | if (this._gaps) {
  47 |     results = s.split(this._pattern);
> 48 |     return (this.discardEmpty) ? _.without(results,'',' ') : results;
     | ^  49 | } else {
  50 |     return s.match(this._pattern);
  51 | }
View compiled
App.componentWillMount
src/App.js:183
  180 | componentWillMount() {
  181 |   var natural = require('natural');
  182 |   var tokenizer = new natural.WordTokenizer();
> 183 |   console.log(tokenizer.tokenize("your dog has fleas."));
      | ^  184 | 
  185 |   var SpeechRecognition = window.SpeechRecognition || window.webkitSpeechRecognition;
  186 |   var SpeechGrammarList = window.SpeechGrammarList || window.webkitSpeechGrammarList;
```

Is there another way I should be installing Lodash? Thanks for any pointers!

**Comment by danielgrigsbyVLT** [comment]
Looks like this is caused by an error in regexp_tokenizer.js which is requiring underscore like this:
_ = require('underscore')._;

Same typo in treebank_word_tokenizer.js
_ = require('underscore')._;

**Comment by Hugo-ter-Doest** [comment]
That is not a typo or error. It works like expected when I run the tests.

**Comment by danielgrigsbyVLT** [comment]
@Hugo-ter-Doest I did install underscore, and still had to remove the ._ at the end of _ = require('underscore')._;

**Comment by chrisrzhou** [comment]
Seeing this issue in `create-react-app` and not sure how to resolve it.

**Comment by Hugo-ter-Doest** [comment]
Seems the same issue as #534

**Comment by Hugo-ter-Doest** [comment]
Solved in #537
